.gitlab/common/test_infra_version.yml

```diff
@@ -4,4 +4,4 @@ variables:
   # and check the job creating the image to make sure you have the right SHA prefix
   TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX: ""
   # Make sure to update test-infra-definitions version in go.mod as well
-  TEST_INFRA_DEFINITIONS_BUILDIMAGES: 4b4112f5f64d
+  TEST_INFRA_DEFINITIONS_BUILDIMAGES: 10486aec63a8
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Recommend bumping `test-infra-definitions` through [the automated bump PR](https://github.com/DataDog/datadog-agent/pull/32321), the advantage is it contains the diff, which ease investigations during incidents. You can also cherry-pick the bump commit  and copy the PR description. We'll probably merge the bump PR before US biz hours today, you can rebase on main after that